css/src/components/card.scss

```diff
@@ -116,7 +116,7 @@ $card-column-gap: $spacer-5 !default;
 		margin: 0;
 		padding: 0;
 		border: none;
-		background-color: $card-background;
+		background-color: transparent;
```

Comment:
no issue with this one, but was there a reason for this having an explicit background? Focus perhaps? 

Also, is this equivalent to no rule? i.e. can this line be removed?

Reply:
Worth mentioning that the `transparent` keyword can show up as solid colors in forced colors mode - we should test and make sure that doesn't cause any issues.

Reply:
Ah, just checked for myself - it looks like Sass might be converting the `transparent` keyword into `#0000`, rather than sticking with CSS's actual `transparent` keyword. That's weird and unintuitive to me, but regardless, testing with forced colors mode for myself showed it seemed to work fine.

Reply:
Updated! Removed the rule completely